### ISSUE TITLE
[Bug :bug:]: ModuleNotFoundError: No module named 'certifi'

### ISSUE THREAD

**Issue by alexmojaki** [SeleniumHQ/selenium#10953]
### What happened?

Error in [this build](https://github.com/alexmojaki/pyodide-worker-runner/runs/7885917987?check_suite_focus=true).

### How can we reproduce the issue?

```shell
from selenium import webdriver
```

### Relevant log output

```shell
test.py:8: in <module>
    from selenium import webdriver
/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/selenium/webdriver/__init__.py:18: in <module>
    from .firefox.webdriver import WebDriver as Firefox  # noqa
/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/selenium/webdriver/firefox/webdriver.py:26: in <module>
    from selenium.webdriver.remote.webdriver import WebDriver as RemoteWebDriver
/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:52: in <module>
    from .remote_connection import RemoteConnection
/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py:25: in <module>
    import certifi
E   ModuleNotFoundError: No module named 'certifi'
```

### Operating System

Ubuntu

### Selenium version

Python 4.4.1

### What are the browser(s) and version(s) where you see this issue?

None

### What are the browser driver(s) and version(s) where you see this issue?

None

**Comment by symonk** [comment]
sorry @alexmojaki tyvm for reporting; how tests didn't flag this is unfortunate..

**Comment by symonk** [comment]
@AutomatedTester whatever we are testing with vs whatever we are building to deploy to pypi are not equal, CI should of definitely flagged this.  We need to ship a new `whl` inclusive of `certifi` since `urllib3[secure]` bundled it in for us as part of that.

**Comment by AutomatedTester** [comment]
Fixed in https://github.com/SeleniumHQ/selenium/commit/54809a701b9cc0306f7277efd3767b02e71be2e5 and I have released 4.4.2

sorry for the inconvenience !
